packages/shared/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "dist/xstate-tools-shared.cjs.js",
   "license": "MIT",
   "dependencies": {
-    "@statelyai/sky": "0.0.4",
+    "@statelyai/sky": "0.0.7",
```

Comment:
Does the `@xstate-tools/shared` package need to depend on `@statelyai/sky`? 🤔

Reply:
It does not. I have removed it in the latest commit